### PR TITLE
Add Goldendict Tools

### PR DESCRIPTION
**Pull request by KonstantinDjairo**

### CODE REVIEW

**Comment by xiaoyifang** [comment]
add a link in  https://github.com/xiaoyifang/goldendict-ng/blob/staged/website/mkdocs.yml

**Comment by KonstantinDjairo** [comment]
> add a link in https://github.com/xiaoyifang/goldendict-ng/blob/staged/website/mkdocs.yml
> 
> it will be generated in here https://xiaoyifang.github.io/goldendict-ng/
> 
> You can put it in a new section, for example, Extension.

done

File: website/docs/topic_gd-tools.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+please notice that gd-tools does works in windows, and we have an [installer](https://www.mediafire.com/file/h1v7owj7np9j7wg/gd-tools_windows.zip/file) for it, you can install and then come back to the previous instruction.
+And if you're at Gnu Guix, install it from our [channel](https://codeberg.org/hashirama/ajattix) <br><br>
+other features:
+- kanji stroke order: for those who want to know how to write a character
+- image searching
+and much more, please see our list [here](https://codeberg.org/hashirama/gd-tools/src/branch/main/README.md#table-of-contents)
+
+# Misc
+we have a mandarin version of gd-marisa, which relies on mecab (unix only) : <br><br>
+![image](https://codeberg.org/hashirama/gd-tools/raw/branch/main/misc/mandarin.png)
```
Comment (xiaoyifang) [inline website/docs/topic_gd-tools.md:25]: Add a #Note here ,
indicate the readers that this article is create by gd-tools author ,not by gd-ng developers?

Comment (KonstantinDjairo) [reply website/docs/topic_gd-tools.md:25]: alright

File: website/docs/topic_gd-tools.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Notes
+This article was written by 柱間.
```
Comment (xiaoyifang) [inline website/docs/topic_gd-tools.md:28]:
```suggestion
This article was written by 柱間(developer of gd-tools).
```

**Comment by xiaoyifang** [comment]
https://xiaoyifang.github.io/goldendict-ng/topic_gd-tools/

**Pull request merged**